substrate/client/executor/common/src/error.rs

```diff
@@ -150,8 +150,8 @@ pub enum WasmError {
 	Other(String),
 }
 
-impl From<polkavm::ProgramParseError> for WasmError {
-	fn from(error: polkavm::ProgramParseError) -> Self {
+impl From<polkavm_common::program::ProgramParseError> for WasmError {
```

Comment:
This type is reexported from the `polkavm` crate; no need to import `polkavm_common` for it.

Reply:
I can recheck this. I got compilation errors but I'll revisit.